docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-provider.md

```diff
@@ -145,12 +145,12 @@ KubeStellar. They label both florin and guilder with `env=prod`, and
 also label guilder with `extended=yes`.
 
 ```shell
-imw1_space_config="${PWD}/temp-space-config/spaceprovider-default-imw1"
-kubectl-kubestellar-get-config-for-space --space-name imw1 --provider-name default --sm-core-config $SM_CONFIG --space-config-file $imw1_space_config
-KUBECONFIG=$imw1_space_config kubectl kubestellar ensure location florin  loc-name=florin  env=prod --imw imw1
-KUBECONFIG=$imw1_space_config kubectl kubestellar ensure location guilder loc-name=guilder env=prod extended=yes --imw imw1
+IMW1_SPACE_CONFIG="${PWD}/temp-space-config/spaceprovider-default-imw1"
```

Comment:
(1) I would make this filename end with ".kubeconfig", so its meaning is utterly clear. This is a common convention.

(2) How is putting "temp-space-config/" in this filename helping?

(3) How is putting "spaceprovider-default" in this filename helping?

(4) I would put "KUBECONFIG" rather than "CONFIG" in this variable name, also to make its meaning utterly clear.